recipes-app/iot2050setup/files/iot2050setup.py

```diff
@@ -643,13 +643,12 @@ def resize(cls):
 
 
 def main():
-    default_console_login = True
-    # get the ssh releated env
-    ssh_client = os.getenv('SSH_CLIENT')
-    ssh_tty = os.getenv('SSH_TTY')
+    default_console_login = False
+    # get the term releated env
+    tty_client = os.getenv('TERM')
 
-    if ssh_client is not None or ssh_tty is not None:
-        default_console_login = False
+    if tty_client == "vt220":
```

Comment:
Is that type sufficient?

Reply:
```
root@iot2050-debian:~# tty
/dev/ttyS3
```

That works by reading where `/proc/self/fd/0` points to - anything /dev/ttyS or ttyUSB would indicate UART without having to probe all terminal types.